dashboard/src/components/OperatorSummary/OperatorSummary.tsx

```diff
@@ -42,19 +42,15 @@ export default function OperatorSummary() {
   }
   return (
     <div className="left-menu">
-      <section className="left-menu-subsection" aria-labelledby="chartinfo-versions">
-        <h5 className="left-menu-subsection-title" id="chartinfo-versions">
-          Capability Level
-        </h5>
+      <section className="left-menu-subsection" aria-labelledby="operatorinfo-capacitylevel">
```

Comment:
Minor changes in `aria-labelledby` as they seemed to have been copied from the chart view.